<!DOCTYPE html>
<head>
	<meta charset="utf-8">
        <title>Curriculum Vitae -- Cameron Prybol</title>
        <meta name="description" content="">
	<link rel="stylesheet" href="./cv-sans-serif.css">
	<link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,400,300,600' rel='stylesheet' type='text/css'>

</head>

<body>
<section id="header" class="level1">
	<h1>Cameron Prybol</h1>
		<hr>
		<h3>
			email: cameron.prybol@gmail.com | phone: 404 683 7308 | GitHub: @cjprybol<br>
		</h3>
		<hr>
</section>

	<!-- example of an entry
		<p>
			<dt>timeframe</dt>
			<dd>
				<p>
					description
				</p>
				<ul>
					<li>bullet of revelant things related to section</li>
				</ul>

			</dd>
		</p>
	-->

<section id="mission statement" class="level3">
	<h3>Mission Statement</h3>
	<p>
		Advancing our understanding of soilborne microbes and their role in agricultural productivity and ecological regeneration
	</p>
</section>

<section id="genomics-and-data-science" class="level3">
	<h3>Genomics and Data Science</h3>
	<dl>
		<p>
			<dt>High Throughput Genomics</dt>
			<dd>
				<p>Terabyte-scale genomics pipeline development and data processing</p>
				<ul>
					<li>Genome assembly, classification, quantification, variant calling</li>
					<li>Transcriptome assembly and differential expression analysis</li>
					<li>Human, model organism, gut, airborne, and soil microbiome datasets</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>Unix/Linux</dt>
			<dd>
				<p>Data acquisition, cleaning, parallel processing and pipeline automation</p>
				<ul>
					<li>Academic HPC clusters, AWS, GCP</li>
					<li>SLURM & Sun Grid Engine (SGE) job schedulers</li>
					<li>Singularity and Docker containers</li>
					<li>Managing, editing, and parsing of data files using awk, sed, grep, parallel, and other standard unix tools</li>
					<li>Bash Shell scripting for automated analysis pipelines</li>
					<li>Managing File/Folder permissions</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>Python</dt>
			<dd>
				<p>Highly proficienct in use of language and core data analytics/visualization libraries<p>
				<ul>
					<li>Experience: 7 years</li>
					<li>Rosalind Bioinformatic challenge problems (http://rosalind.info/users/cameron.prybol/)</li>
					<li>Recruited by Google after successful completion of foobar challenge</li>
					<li>Data visualization with Matplotlib and Plotly</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>Julia</dt>
			<dd>
				<p>Highly proficienct in language understanding/development and package development/maintenance<p>
				<ul>
					<li>Experience: 4 years</li>
					<li>Language of choice for developing custom bioinformatics software</li>
					<li>Contributor and developer of several core data analytics packages</li>
					<li>Previous member of JuliaData organization</li>
					<li>Data visualization with Plots.jl and custom visualizations with Luxor.jl</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>Open Source Contributions</dt>
			<dd>
				<p>Personal Projects</p>
				<ul>
					<li>reproducibility-via-singularity - 25 stars</li>
					<li>uCSV.jl - 6 stars</li>
					<li>foobar_solutions - 4 stars</li>
				</ul>
			</dd>
			<dd>
				<p>Existing Projects</p>
				<ul>
					<li>JuliaLang/Julia</li>
					<li>JuliaData/DataFrames.jl</li>
					<li>JuliaGraphs/LightGraphs.jl</li>
					<li>sylabs/singularity</li>
					<li>Homebrew/homebrew-science</li>
					<li>JuliaStats/Distributions.jl</li>
					<li>JuliaStats/StatsBase.jl</li>
					<li>JuliaData/CSV.jl</li>
					<li>JuliaStats/DataArrays.jl</li>
					<li>JuliaStats/NullableArrays.jl</li>
					<li>JuliaData/CategoricalArrays.jl</li>
					<li>JuliaData/Missings.jl</li>
				</ul>
			</dd>
		</p>
	</dl>
</section>

<section id="education" class="level3">
	<h3>Education</h3>
		<p>
			<dl>
				<dt>PhD</dt>
				<dd>
					<p>Department of Genetics, Stanford University School of Medicine, Stanford, CA</p>
					<ul>
						<li><em>July 2015 - April 2019</em></li>
						<li>incomplete</li>
					</ul>
				</dd>
			</dl>
		</p>

		<p>
			<dl>
				<dt>Bachelor of Science</dt>
				<dd>
					<p>Biochemistry and Molecular Biology, University of Georgia, Athens, GA<p>
					<ul>
						<li><em>2010-2014</em></li>
						<li>Cum Laude, Graduation with Honors</li>
					</ul>
				</dd>
			</dl>
		<p>
</section>


<section id="research-experience" class="level3">
	<h3>Research Experience</h3>
	<dl>
		<p>
			<dt>
				<em>2016-2019</em>
			</dt>
			<dd>
				<p>Ashley/Synder Labs, Precision Medicine, Department of Genetics, Stanford University</p>
				<ul>
					<li>Probablistic Graph-based Approaches to Genome Assembly, Identification, Analysis, and Quantification</li>
				</ul>
			</dd>
		</p>
		<p>
			<dt>
				<em>2015</em>
			</dt>
			<dd>
				<p>Li Lab, RNA-editing, Department of Genetics, Stanford University -- <em>rotation</em></p>
			</dd>
		</p>
		<p>
			<dt>
				<em>2015</em>
			</dt>
			<dd>
				<p>Montgomery Lab, Human Population Genetics, Department of Genetics, Stanford University -- <em>rotation</em></p>
			</dd>
		</p>
		<p>
			<dt>
				<em>2014-2015</em>
			</dt>
			<dd>
				<p>Lewis lab, Chromatin Function and Regulation, Department of Microbiology, University of Georgia</p>
				<ul>
					<li>Heterochromatin regulation in knockout strain libraries and genetic crosses</li>
					<li>High-throughput genetic screens, DNA sequencing, and bioinformatic analysis</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>
				<em>2012-2013</em>
			</dt>
			<dd>
				<p>Adams lab, BESC biofuels project, Department of Biochemistry, University of Georgia</p>
				<ul>
					<li>Media and growth condition optimization in hyperthermophilic biomass fermentations</li>
					<li>Reaction rates and enzyme kinetics of recombinant synthesis pathways</li>
				</ul>
			</dd>
		</p>

		<p>
			<dt>
				<em>2012-2015</em>
			</dt>
			<dd>
				<p>Un estudio de Lepidoptera en el valle de San Luis (Costa Rica), independent research project</p>
				<ul>
					<li>Investigating phenology, voltinism, and abundance of tropical Lepidoptera</li>
					<li>Acquired funding, developed sampling protocols, trained and managed interns for 2 years</li>
				</ul>
			</dd>
		<p>

		<p>
			</dd>
				<dt>
					<em>2011-2015</em>
				</dt>
				<dd>
					<p>Pickering lab, Odum School of Ecology, University of Georgia<p>
					<ul>
						<li>Studying large-scale driving factors of Lepidopteran phenology, voltinism, and abundance across latitudinal gradients</li>
						<li>Developed species identification guides using robust diagnostic characteristics</li>
						<li>Manage databases and webpages in Unix (Sun Solaris) environment</li>
						<li>Identification of North, Central, and South American macro-Lepidoptera</li>
					</ul>
			</dd>
		</p>
	</dl>
</section>


<section id="publications" class="level3">
	<h3>Publications</h3>

		<dt><em>2019</em></dt>
		<dd>
			<p><b>Prybol, CJ</b>, Jiang, C, Snyder, M. Probablistic Graph-based Genome Analystics and Metagenomic Applications.
			<em>manuscript in prep</em></p>
		</dd>

		<dt><em>2019</em></dt>
		<dd>
			<p>Frésard, L. et al. Identification of rare-disease genes using blood transcriptome sequencing and large control cohorts. Nature Medicine 25, 911–919 (2019).</p>
		</dd>

		<dt><em>2019</em></dt>
		<dd>
			<p>Zastrow, DB, et al. A toolkit for genetics providers in follow‐up of patients with non‐diagnostic exome sequencing. Journal of genetic counseling 28.2 (2019): 213-228.</p>
		</dd>

		<dt><em>2017</em></dt>
		<dd>
			<p>Sochat VV, <b>Prybol CJ</b>, Kurtzer GM (2017) Enhancing reproducibility in scientific computing: Metrics and registry for Singularity containers. PLoS ONE 12(11)</p>
		</dd>


		<dt><em>2015</em></dt>
		<dd>
			<p>Basenko, EY, Sasaki, T, Ji, L, <b>Prybol, CJ</b>, Burckhardt, RM, Schmitz, RJ, &amp; Lewis, ZA (2015). Genome-wide redistribution of H3K27me3 is linked to genotoxic stress and defective growth. Proceedings of the National Academy of Sciences.</p>
		</dd>

		<dt><em>2014</em></dt>
		<dd>
			<p>Basen, M, Rhaesa, AM, Kataeva, I, <b>Prybol, CJ</b>, Scott, IM, Poole, FL, &amp; Adams, MWW (2014). Degradation of high loads of crystalline cellulose and of unpretreated plant biomass by the thermophilic bacterium Caldicellulosiruptor bescii. Bioresource Technology, 152, 384-392. http://doi.org/10.1016/j.biortech.2013.11.024</p>
		</dd>

		<dt><em>2014</em></dt>
		<dd>
			<p>Basen, M, Schut, GJ, Nguyen, DM, Lipscomb, GL, Benn, RA, <b>Prybol, CJ</b>, et al. (2014). Single gene insertion drives bioalcohol production by a thermophilic archaeon. Proceedings of the National Academy of Sciences, 111(49), 17618-17623. http://doi.org/10.1073/pnas.1413789111
		</dd>
</section>

<section id="presentations-and-conferences" class="level3">
	<h3>Presentations and Conferences</h3>
	<dl>
		<p>
			<dt><em>March 2019</em></dt>
			<dd>
				<p>JIMB/NIST Trainee Symposium</p>
			</dd>
		</p>

		<p>
			<dt><em>April 2016</em></dt>
			<dd>
				<p>NHGRI Career Development Symposium</p>
			</dd>
		</p>

		<p>
			<dt><em>April 2013</em></dt>
			<dd>
				<p>UGA CURO symposium</p>
			</dd>
		</p>

		<p>
			<dt> <em>June 2012</em> </dt>
			<dd>
				<p>Costa Rica Live</p>
			</dd>
		</p>

		<p>
			<dt><em>March 2012</em></dt>
			<dd>
				<p>UGA Discussion Circles</p>
			</dd>
		</p>
</section>


<section id="scholarships-grants-and-awards" class="level3">
	<h3>Scholarships, Grants, Awards, and Funding</h3>
	<dl>
		<p>
			<dt> <em>2016-2019</em> </dt>
			<dd>
				<p>Joint Initiative for Metrology in Biology</p>
			</dd>
		</p>

		<p>
			<dt> <em>2015-2016</em> </dt>
			<dd>
				<p>Stanford Genome Training Program</p>
			</dd>
		</p>

		<p>
			<dt> <em>2015-2015</em> </dt>
			<dd>
				<p>Stanford ADVANCE Summer Institute</p>
			</dd>
		</p>

		<p>
			<dt><em>2013</em></dt>
			<dd>
				<p>UGA Honors International Scholar Program</p>
			</dd>
		</p>

		<p>
			<dt> <em>2012</em> </dt>
			<dd>
				<p>UGA CURO Summer Fellowship</p>
			</dd>
		</p>

		<p>
			<dt><em>2010-2014</em></dt>
			<dd>
				<p>Hope Scholarship</p>
			</dd>
		</p>
</section>

</body>
</html>
